index.js

```diff
@@ -1,12 +1,7 @@
-
 const register = function (server, options, next) {
-  console.log('registering tags');
-  server.route({
-    method: 'GET',
-    path: '/test',
-    handler: function (request, reply) {
-        return reply('ok');
-    }
+  server.ext('onPreAuth', function(request, reply) {
```

Comment:
It will be nice to add some docs which explain what is server.ext and how to we use it to add more functions to the server.